.gitignore

```diff
@@ -35,3 +35,4 @@ case_studies/*/data/
 venv
 nohup.out
 multirun
+case_studies/dc2/*.pkl
```

Comment:
Could you create a .gitignore file within case_studies/dc2 rather than modifying the top-level one?

Reply:
Fix it in commit 1987fcc